app/collins/models/shared/IpAddressable.scala

```diff
@@ -127,11 +127,11 @@ trait IpAddressStorage[T <: IpAddressable] extends Schema with AnormAdapter[T] w
   * For a range 0L..20L, used addresses List(17,18,19,20), the result will be None (allocate from beginning)
   */
   protected def getCurrentLowestLocalMaxAddress(calc: IpAddressCalc)(implicit scope: Option[String]): Option[Long] = inTransaction {
-    val minAddress = calc.minAddressAsLong
+    val startAddress = calc.startAddressAsLong
```

Comment:
@roymarantz hmmmm true.... at the very least we should do something as part of #446 - either more intelligent default values, or better documentation around what will happen in each scenario.  Or we just remove the defaults, and make people specify their start address and gateway, to avoid all the problems with assumed default values.